pandera/schemas.py

```diff
@@ -32,6 +32,19 @@
 
 PandasDtypeInputTypes = Union[str, type, PandasDtype, PandasExtensionType]
 
+if pd.__version__ < "1.0.0":
```

Comment:
might be more robust to use `version.packaging` as in [here](https://github.com/pandera-dev/pandera/blob/master/tests/test_dtypes.py#L21) and [here](https://github.com/pandera-dev/pandera/blob/master/tests/test_dtypes.py#L120-L123) as opposed to relying on string comparison